packages/ui/react-ui/src/components/Dialog/Dialog.tsx

```
@@ -140,7 +141,11 @@ type DialogHeaderProps = ThemedClassName<PropsWithChildren> & { srOnly?: boolean
 const DialogHeader: ForwardRefExoticComponent<DialogTitleProps> = forwardRef<HTMLHeadingElement, DialogTitleProps>(
   ({ classNames, srOnly, ...props }, forwardedRef) => {
     const { tx } = useThemeContext();
-    return <div {...props} role='header' className={tx('dialog.header', { srOnly }, classNames)} ref={forwardedRef} />;
+    return (
+      <Container.Segment>
+        <div role='header' {...props} className={tx('dialog.header', { srOnly }, [classNames])} ref={forwardedRef} />
```

Biome (2.4.4)

[error] Enforce that elements with ARIA roles must use a valid, non-abstract ARIA role.

(lint/a11y/useValidAriaRole)